Automation/Redis_Cache/deploy.ps1

```diff
@@ -0,0 +1,27 @@
+
+# to run this, navigate to the Powershell  /Automation/Redis_Cache/deploy.ps1
+
+
+param(
+        [string] [Parameter(Mandatory=$true)] $SubscriptionId,
+        [string] [Parameter(Mandatory=$true)] $ResourceGroupName,
+        [string] [Parameter(Mandatory=$true)] $Location,
+        [string] [Parameter(Mandatory=$true)] $Redis_nsc_redis_dev_usw2_thursday_name
+
+      )
+
+$pathToRedisTemplate = "/Users/student/Documents/GitHub/ad440-winter2021-thursday-repo/Automation/Redis_Cache/template.json"   
+
```

Comment:
I think this template file path is specific to your machine.

Reply:
Yeah, the you can direct the tester to navigate to this folder and then just use the relative path.

Reply:
Thanks for catching that!